bang/models.py

```diff
@@ -1511,133 +1517,120 @@ def __unicode__(self):
-
-    @property
-    def formatted_description(self):
-        template = (
-            _('{affected_members} members get +{effect_value} {affected_stat} points if life is above {life}')
-            if self.depends_on_life
-            else _('{affected_members} members get +{effect_value} {affected_stat} points')
-        )
-        return template.format(**{
-            variable: unicode(getattr(self, variable, ''))
-            for variable in (templateVariables(template) or [])
-        })
+        formatted_name=''
+        if unicode(self.member) != u'None':
+            if self.type in ['studio', 'poster', 'entrance']:
+                if self.type == 'studio' and self.instrument not in [None, 'mic']:
+                    formatted_name+=unicode(_('{name}\'s').format(name=unicode(self.member.first_name)))
+                else:
+                    formatted_name+=unicode(self.member.t_band)
+        if self.t_name:
+            formatted_name +=' ' + unicode(self.t_name)
+        if self.instrument not in ['other', None]:
+            formatted_name+=' ' + unicode(self.t_instrument)
+        return formatted_name or ''
+
+    @property
+    def affected(self):
+        string = ''
+        if self.i_attribute != None:
+            string += unicode(self.t_attribute)
+        if self.member != None:
+            if string != '':
+                string +=' '
+            string += self.member.t_band
+        return string if string != '' else None
+
+    @property
+    def stat(self):
+        if self.i_boost_stat is not None:
+            return unicode(self.t_boost_stat).lower()
+        return unicode(_('All')).lower()
+
+    def formatted_description(self, level=1):
+        value = self.values.split() if self.values != None else ''
+        if level-1 > len(value)-1:
+            value='???'
+        else:
+            value=value[level-1]
+        life = self.lifes.split() if self.lifes != None else ''
+        if level-1 > len(life)-1:
+            life='???'
+        else:
+            life=life[level-1]
+        if self.lifes != None:
+            if self.affected != None:
+                return _('Restores life by {life} and {affected} members get a {value} boost on {stat} Stats').format(
+                    life=life, affected=self.affected, value=value, stat=self.stat)
+            return _('Restores life by {life} and {value} boost on {stat} Stats').format(
+                life=life, value=value, stat=self.stat)
+        elif self.affected != None:
+            return _('{affected} members get a {value} boost on {stat} Stats').format(
+                affected=self.affected, value=value, stat=self.stat)
+        elif self.values != None:
+            return _('{value} boost on {stat} Stats').format(value=value, stat=self.stat)
+        return u''                 
 
     @property
     def max_level(self):
-        return 6 if self.type == 'instrument_per_band' else 5
+        return 6 if self.type == 'studio' else 5
```

Comment:
I think this should be made into an editable field, or at least based on the number of values provided.